manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -375,6 +381,8 @@ jobs:
             uaa: {}
       metron_agent:
         zone: z1
+      route_registrar:
+        routes: []
```

Comment:
As far as I know there is no way to install a package without a job. Unless we create a new fake job in a new bosh-release.
The issue [was reported](https://github.com/cloudfoundry/cf-release/issues/950). Apparently it was fixed in the develop branch, so we can remove the workaround when we upgrade to the right CF version.